src/main/java/tds/exam/results/configuration/ExamResultsTransmitterServiceProperties.java

```diff
@@ -13,7 +13,10 @@
     private String tisCallbackUrl = "";
     private boolean validateTrtXml = false;
     private boolean sendToTis = false;
-    private boolean retryOnError = true;
+    private long retryInitialInterval = 1000;
+    private double retryIntervalMultiplier = 2;
+    private long retryMaxInterval = 5000;
+    private int retryAmount = 3;
```

Comment:
Why have both a retryAmount and a retryMaxInterval?  Don't both function as a "this ain't gonna work" cutoff?